Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1

```diff
@@ -92,7 +99,7 @@ function Set-TargetResource
         throw "This resource cannot delete Managed Properties. Please make sure you set its Ensure value to Present."
```

Comment:
Please update error message

Reply:
`throw "O365AdminAuditLogConfig configurations MUST specify Ensure value of 'Present'"`
Better?